packages/harmony/src/components/layout/Flex/Flex.tsx

```diff
@@ -1,11 +1,16 @@
+import isPropValid from '@emotion/is-prop-valid'
 import styled from '@emotion/styled'
 
 import { Box } from '../Box'
 
 import type { FlexProps } from './types'
 
+const invalidProps = ['alignItems', 'direction', 'wrap']
```

Comment:
these are just the props flex has but not box?

what about justifyContent, etc.? are those caught by `isPropValid` but these aren't?

Reply:
yeah i think so, this was me inspecting the dom on my page, but let me add justifyContent and see what happens :)